composer.json

```diff
@@ -18,21 +18,25 @@
     }
   ],
   "require": {
-    "php": ">=7",
+    "php": "7.4 - 8.1",
```

Comment:
Should this be `>= 7.4 || >= 8`?

Reply:
I find the hyphenated range easier to read, and I like to intentionally smoke test before bumping the upper bound when the next version of PHP is released.

I'm open to changing it though. I haven't worked with WP or PHP for a while.

Reply:
> I find the hyphenated range easier to read, and I like to intentionally smoke test before bumping the upper bound when the next version of PHP is released.

Cool, works for me 😊 I don't have a strong opinion about it.